packages/openblas/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: openblas
+  version: de4d5646eb1ac6a0e663a4c9bc797dad8226aedb
+
+source:
+  sha256: 148a959b61ffa4d77e810ceb3114765501d70c982cbb302a00954b81bc918e46
+  url: https://github.com/xianyi/OpenBLAS/archive/de4d5646eb1ac6a0e663a4c9bc797dad8226aedb.zip
+  patches:
+    - patches/0001-Add-Wno-implicit-function-declaration-flag.patch
+
+build:
+  type: shared_library
+  script: |
+    # seems like .zip does not maintain executable flags, need to reset these
+    chmod u+x c_check
+    chmod u+x f_check
+    chmod u+x exports/gensymbol
+    make CC=emcc HOSTCC=gcc TARGET=RISCV64_GENERIC NOFORTRAN=1 USE_THREAD=0
+    mkdir -p dist
+    cp libopenblas.so dist/
+    mkdir -p ${WASM_LIBRARY_DIR}/{lib,include}
+    # TODO do I need include as in CLAPACK?
```

Comment:
I would have guessed yes, but you can also probably not include it and see if there are any related include errors as done now.